apps/gateway-ui/src/languages/en.json

```diff
@@ -36,11 +39,17 @@
     "all": "All"
   },
   "info-tab": {
-    "tab-header": "Information",
-    "balance": "Balance:"
+    "tab_header": "Node ID",
+    "node_id_link": "View on 1ml.com"
```

Comment:
nit: link used points to amboss.com - which is fine? I suggest we update the text